### DISCUSSION TITLE
What do you think about Tanstack's example style?

### DISCUSSION THREAD

**Discussion by githorse** [TanStack/table#4849] [Polls]
In the Tanstack Table docs, an example takes me directly to an embedded CodeSandbox, like [this](https://tanstack.com/table/v8/docs/examples/react/column-pinning). (The [examples are all broken at the moment](https://github.com/TanStack/table/issues/4844), but that's a different problem.) That means that, in order to see the sample code, I have to:

1. Load the page, which takes a couple of seconds to spin up the sandbox.
2. Click through into the `src` folder, look for the `main.tsx` file or whatever, and click on it.
3. Browser the code in the tiny vertical window between the sidebar and the grid.

Compare this with, say, [MUI's docs](https://mui.com/x/react-data-grid/#mit-version), or [Material React Table's](https://www.material-react-table.com/docs/guides/sorting#sort-direction). Here the code is already on the screen for me to browse (as is the component that results from that code), no loading time required. If I want to *edit* that code and play with it -- which happens much more rarely than just reading it -- there's a convenient `[Open in CodeSandbox]` link I can use.

My preference is probably pretty clear from the way I've phrased this: the second (MUI) style wins, hands down. But what do other people think?

**Comment by wxl2578** [comment]
The documentation of this library is bad, and there are many questions that no one responds to. I don’t know why so many people use it.

**Comment by KevinVandy** [comment]
I keep seeing reports of the code sandboxes not working for people, but whenever I check them out in the docs, they are all working out perfectly fine

<img width="1558" alt="image" src="https://github.com/TanStack/table/assets/28243511/575b30cf-497a-4259-9681-08cf3bc1c891">

However, maybe we out to switch over to Stackblitz like the Query docs
